.github/workflows/release.yaml

```diff
@@ -29,8 +29,8 @@ jobs:
 
       - name: Add dependency chart repos
         run: |
-          helm repo add stable https://kubernetes-charts.storage.googleapis.com/
-          helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
+          helm repo add stable https://charts.helm.sh/stable/
```

Comment:
do we need these at all? can't we get rid of this stage?

Reply:
I have absolutely no idea - however I'm able to create a package without them - I guess author of that action - @torstenwalter - is the best person to ask.

Reply:
I would remove them

Reply:
done